.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
```

Comment:
We prefer to keep all the configs in "setup.cfg" as we did in keras-tuner.

Reply:
Do you have commented the right line?

Reply:
I think so. [This](https://github.com/keras-team/keras-tuner/blob/master/setup.cfg#L30-L43) is where we keep the flake8 configs.

Reply:
Ok done. 
When https://github.com/PyCQA/flake8/issues/234 it will be ready probably we could unify `flake` config on the same file.

Reply:
Do you want to copy the whole Keras `setup.cfg` here? As I've just mapped the current `args` that we already had  in this repo to config files.

Reply:
I think it is good for now. We can move it later.